### PR TITLE
fix remix apps being broken on latest vite alpha release

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: README.md
Comment (dario-piotrowicz) [inline README.md:37]: I forgot to remove this when I removed the patch

File: examples/remix-app-node-vm/README.md
Comment (dario-piotrowicz) [inline examples/remix-app-node-vm/README.md:22]: I've removed this when I removed the patch, also setting `mainField` is no longer necessary without the patch

File: package.json
Comment (dario-piotrowicz) [inline package.json]: TODO: I need to do another release of our remix for and link to that before merging

(but before I can release that we'd need to release the provider packages with the changes present in this PR and use those in the remix fork)

File: package.json
Comment (dario-piotrowicz) [inline package.json]: this points to the latest vite version from their main branch

we can use this or wait for their next release and keep using more proper/stable npm releases

File: packages/vite-environment-provider-cloudflare/src/index.ts
Comment (dario-piotrowicz) [inline packages/vite-environment-provider-cloudflare/src/index.ts:121]: TODO: do the same sort of thing for the node environments

Comment (dario-piotrowicz) [reply packages/vite-environment-provider-cloudflare/src/index.ts:121]: Done

File: package.json
Comment (dario-piotrowicz) [reply package.json]: I've released the updated version of my remix fork and as suggested by @jamesopstad I've removed the requirement for the environment providers 👍

File: package.json
Comment (jamesopstad) [reply package.json]: I just worry that someone else might look at the project and then it could be broken because of a configuration change. I think bumping the version numbers manually and checking for changes might be better.

**Review by jamesopstad — APPROVED** [review]
Decide what you think is best about the Vite version but otherwise looks good.

File: package.json
Comment (dario-piotrowicz) [reply package.json]: let's just go with `6.0.0-beta.0` 🙂

**Comment by dario-piotrowicz** [comment]
Thanks @jamesopstad 🙂 

I've bumped vite to `6.0.0-beta.0`, I think this is more stable/predictable than staying on the latest version from main, so this for me is slightly preferred so that things can't break under our feet easily, let's stick with this since you also don't seem to have a strong preference either way 🙂👍

PS: using `https://pkg.pr.new/vite@main` could be quite handy in indeed helping us find out sooner when something changes and can potentially break us, we can always change use this later 🙂

**Pull request merged**
